documentation/md/docs/Getting-Started.md

```diff
@@ -92,4 +92,42 @@ const getSession = neogma.getSession; // @see [Sessions](./Sessions)
 const modelsByName = neogma.modelsByName; // @see [Defining a Model](./Models/Defining-a-Model)
 ```
 
+## Temp DB's
```

Comment:
I'd prefer if they were in their own section (i.e. `Temporary-Databases.md`) instead of here, as it adds a lot of noise that most users won't need. Be sure to also include it to the table of contents (`_toc.md` file).

Reply:
Fair point. Will do.